docs/internal/contributing/contributing-to-helm-chart.md

```diff
@@ -36,6 +36,12 @@ Install [ct](https://github.com/helm/chart-testing) and run
 ct lint --config operations/helm/ct.yaml --charts operations/helm/charts/mimir-distributed
 ```
 
+Or use Docker without installing dependencies and run
+
+```bash
+docker run --rm -it --volume $(pwd):/data quay.io/helmpack/chart-testing:latest sh -c "ct lint --all --debug --chart-dirs /data/operations/helm/charts --check-version-increment false --config /data/operations/helm/ct.yaml"
```

Comment:
I think we should just replace the previous instructions. This requires installing less things.

also you don't need the interactive and tty flags

```suggestion
docker run --rm --volume $(pwd):/data quay.io/helmpack/chart-testing:latest sh -c "ct lint --all --debug --chart-dirs /data/operations/helm/charts --check-version-increment false --config /data/operations/helm/ct.yaml"
```

Reply:
updated command to not run as root and removed `-it`